modules/identity-provider-add.adoc

```diff
@@ -22,15 +22,10 @@ users can authenticate.
 
 * Create an {product-title} cluster.
 * Create the Custom Resource (CR) for your identity providers.
+* You must be logged in as an administrator.
 
 .Procedure
 
-. Log in to the cluster as the `kube-admin` user:
```

Comment:
Do we make it clear enough that the kubeadmin is the user that you need to do this with for your initial IDP? I might make the prereq reference the kubeadmin by adding something like "For new clusters, the default admin user is `kubeadmin`."

Reply:
Part of the goal here was to remove references to the `kubeadmin` user. If they haven't defined an identity provider then they should be logging in as the kubeadmin user, especially since we don't advertise `oc create user` as an option.